packages/@okta/vuepress-site/docs/reference/api/brands/index.md

```diff
@@ -1095,6 +1095,936 @@ curl -v -X DELETE \
 HTTP/1.1 204 No Content
 ```
 
+## Email template operations
+
+The Email Templates API has the following operations:
+
+* [List email templates](#list-email-templates)
+* [Get email template](#get-email-template)
+* [Get email template default content](#get-email-template-default-content)
+* [Preview email template default content](#preview-email-template-default-content)
+* [List email customizations](#list-email-customizations)
+* [Create email customization](#create-email-customization)
+* [Delete all email customizations](#delete-all-email-customizations)
+* [Get email customization](#get-email-customization)
+* [Update email customization](#update-email-customization)
+* [Delete email customization](#delete-email-customization)
+* [Preview email customization](#preview-email-customization)
+* [Send test email](#send-test-email)
+
+### List email templates
+
+<ApiOperation method="get" url="/api/v1/brands/${brandId}/templates/email" />
+
+Lists all supported email templates.
+
+#### Request path parameters
+
+| Parameter      | Type        | Description            |
+| -------------- | ----------- | ---------------------- |
+| `brandId`      | String      | ID of a Brand          |
+
+#### Response body
+
+A [paginated](/docs/reference/core-okta-api/#pagination) list of [Email Template](#email-template) resources.
+
+Passing an invalid `brandId` returns a `404 Not Found` with error code `E0000007`.
+
+#### Use examples
```

Comment:
I know that we use this format elsewhere in the Brands API doc, but I only see it used in one other API reference. I think it's not needed.
```suggestion
```